package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-client",
   "description": "FeedHenry MBaaS Client",
-  "version": "0.15.0",
+  "version": "0.15.1",
   "author": "FeedHenry",
   "main": "index.js",
   "directories": {
     "lib": "lib"
   },
   "engines": {
-    "node": ">=0.8 < 0.11"
+    "node": "4.4"
```

Comment:
This change here instructs bob to build and run the tests with NodeJS 4 now. I think that's fine, but it's just something to be aware of -- given that this module is used by fh-mbaas-api, and could be used in someones cloud app that they deploy and set the NodeJS 0.10 runtime. If we use any NodeJS 4 features that aren't compatible with NodeJS 0.10, it might be broken for them.

Maybe the absolute best thing to do for these modules that are used in fh-mbaas-api, is to build and run tests with both versions of nodejs, until nodejs 0.10 is no longer available as a runtime option in the studio.